README.md

```diff
@@ -1,2 +1,52 @@
 # FitTracker
 Service runs on port 8080
+
+## Containers
```

Comment:
Let's only put useful info in our readme. What I'd like to see is this:

# FitTracker

App for tracking your weight and calorie intake.

## Local setup

### Prerequisites

List of stuff that should be installed in order to run the project locally (java 21, maven, docker) and setup steps (if there are any).

### Database

Instructions step by step, e.g. use this command to run containers, go to pgAdmin on http://localhost:5050, login with these credentials, add new server, type this and that, open query tool, run `SELECT 1` to verify db connection.

### SpringBoot App

Install with this maven command, run with this maven command. Call http://localhost:8080/health endpoint to verify it's up, mention the expected response.